### DISCUSSION TITLE
Any chance of bringing AutoPlay back?

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1777] [Q&A] [answered]
My apologise if this has already been reported or talked about but I noticed that the AutoPlay functionality has been removed and that causes streams to break because live stream need the autoplay function to playback.

Also, I have no way of telling whats been removed or whats been added.

**Comment by networkException** [comment, marked as answer]
In the last several months at least there have been no changes to anything related to AutoPlay

**Reply by ghost** [reply]
Hmmm, I seem to not see it on my installation of Ungoogled Chromium

**Reply by networkException** [reply]
Can you describe what you have tested exactly, where you installed ungoogled-chromium from and since which version you have noticed a change in behavior?

**Reply by ghost** [reply]
Currently I have Version 96.0.4664.45 installed and I believe from Version 89.0.4389.114-1.1 till 96.0.4664.45 it has disappeared.

**Reply by ghost** [reply]
Also, I have installed Ungoogled Chromium in Portable Mode on my External Harddrive and I am testing this on Both Twitch, TrovoLive, and GlimeshTV

**Reply by networkException** [reply]
What exactly does not work about Twitch? I don't have any issues when just navigating to a stream.

https://user-images.githubusercontent.com/42888162/146830540-702ec09f-8925-4cb3-b677-9d234fff6bf4.mp4

**Reply by ghost** [reply]
I am trying to enable it in the Site Settings but its not listed anywhere

https://user-images.githubusercontent.com/37266538/146830838-cdf4c751-dc0c-4f94-b2e0-b971314f3a7f.mp4

**Reply by ghost** [reply]
As you can see here
![UkF6fXZHY9](https://user-images.githubusercontent.com/37266538/146831014-008eac06-7c47-4062-8d8f-c6e263714494.png)

**Reply by networkException** [reply]
I don't know what put you under the impression that AutoPlay should be configurable in more recent versions of chromium but it's not the case and this project does currently not change anything about that.

**Reply by ghost** [reply]
Ahh alright. Sorry for the inconvenience. Feel free to close this discussion.
